aggregate/site/category.xml

```diff
@@ -285,6 +285,7 @@
   <feature id="org.jboss.tools.quarkus.feature">
     <category name="CoreTools" />
   </feature>
+  <bundle id="com.redhat.quarkus.jdt.core"/>
```

Comment:
I think this should be included into the Quarkus feature so this should be done in jbosstools-quarkus

Reply:
see https://github.com/jbosstools/jbosstools-quarkus/pull/43

Reply:
resolved since https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Devstudio/view/devstudio_master/job/jbosstools-quarkus_master/19/